src/style.css

```
@@ -0,0 +1,35 @@
+
+body {
```

Don't add redundant empty lines
```suggestion
body {
```